dask/dataframe/tests/test_shuffle.py

```diff
@@ -1720,8 +1720,16 @@ def test_sort_values_bool_ascending():
     ddf = dd.from_pandas(df, npartitions=10)
 
     # attempt to sort with list of ascending booleans
-    with pytest.raises(NotImplementedError):
+    with pytest.raises(ValueError, match="length"):
         ddf.sort_values(by="a", ascending=[True, False])
```

Comment:
This added case currently works in `dask-expr`, and I'd like to change that.

```suggestion
    with pytest.raises(ValueError, match="length"):
        ddf.sort_values(by="a", ascending=[True, False])

    with pytest.raises(ValueError, match="length"):
        ddf.sort_values(by=["a", "b"], ascending=[True])
```